clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -593,6 +1088,10 @@ Error linkBitcodeFiles(SmallVectorImpl<OffloadFile> &InputFiles,
   const llvm::Triple Triple(Args.getLastArgValue(OPT_triple_EQ));
   StringRef Arch = Args.getLastArgValue(OPT_arch_EQ);
 
+  // Early exit for SPIR targets
```

Comment:
Do I understand correctly, that this is rather a temporary thing until we enable thinLTO support?
Essentially, as we increase level of SYCL integration into the tool, we will re-use more of existing routines, right?

Reply:
Precisely. There are several blockers. But we intend to use existing routines (with minor mods if needed).

Thanks